composer.json

```diff
@@ -16,10 +16,10 @@
         }
     ],
     "require": {
-        "php": "^7.3|^8.0",
+        "php": "^7.3|^8.0|^8.1",
         "ext-json": "*",
         "guzzlehttp/guzzle": "^6.0|^7.0",
-        "illuminate/support": "^7.0|^8.0"
+        "illuminate/support": "^7.0|^8.0|^9.0"
```

Comment:
@developh 
The `illuminate/support` update as well as updates for some other dependencies are in the latest updates to the original package.
The php update is not.

I've created a draft PR for Grayloon's updates.
https://github.com/interiordefine/laravel-magento-2/pull/6